src/plugins/shared_ux/public/components/toolbar/solution_toolbar/quick_group_buttons/quick_group.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import {
+  EuiButtonGroup,
+  htmlIdGenerator,
+  EuiButtonGroupOptionProps,
+  useEuiTheme,
+  IconType,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { css } from '@emotion/react';
+
+const strings = {
+  getAriaButtonLabel: (createType: string) =>
+    i18n.translate('sharedUX.solutionToolbar.quickButton.ariaButtonLabel', {
+      defaultMessage: `Create new {createType}`,
+      values: {
+        createType,
+      },
+    }),
+  getLegend: () =>
+    i18n.translate('sharedUX.solutionToolbar.quickButton.legendLabel', {
+      defaultMessage: 'Quick create',
+    }),
+};
+
+export interface QuickButtonProps {
+  createType: string;
```

Comment:
nit: pretty sure this is a carry-over from Dashboard, where these buttons were strictly for creating things.  This isn't a restriction for this component, so consider a different prop name.

You rename this to `label` on 50... probably makes sense to just name this `label`...?

Reply:
Sounds good a014375 for changes